README.rst

```diff
@@ -9,5 +9,48 @@ This is the ``pulp_rpm`` Plugin for `Pulp Project
 content types, similar to the ``pulp_rpm`` plugin for Pulp 2.
 
 For more information, please see the `documentation
-<http://pulp-rpm.readthedocs.io/en/latest/>`_ or the `Pulp project page
+<https://docs.pulpproject.org/pulp_rpm/>`_ or the `Pulp project page
 <https://pulpproject.org>`_.
+
+
+How to File an Issue
+--------------------
+
+`New pulp_rpm issue <https://pulp.plan.io/projects/pulp_rpm/issues/new>`_.
+
+.. warning::
+  Is this security related? If so, please follow the `Security Disclosures <https://docs.pulpproject.org/pulpcore/bugs-features.html#security-bugs>`_ procedure.
+
+Please set **only the fields in this table**. See `Redmine Fields <https://docs.pulpproject.org/pulpcore/bugs-features.html#redmine-fields>`_ for more detailed
+descriptions of all the fields and how they are used.
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+   :align: center
+
+   * - Field
+     - Instructions
+
+   * - Tracker
+     - For a bug, select ``Issue``, for a feature-request, choose ``Story``,
+       for a backport request, choose ``Backport``.
+
+   * - Subject
+     - Strive to be specific and concise.
+
+   * - Description
+     - This is the most important part! Please see `Description Field <https://docs.pulpproject.org/pulpcore/bugs-features.html#issue-description>`_.
+
+   * - Category
```

Comment:
I'd remove this field, from the list. Users often can't get it right, and mostly because we don't have good clear definitions for each category and we just know internally how we use it.